environment.M1.yml

```diff
@@ -0,0 +1,9 @@
+name: sdk
+channels:
+  - default
+  - apple
+  - conda-forge
+dependencies:
+  - python=3.8.13
```

Comment:
Should we instead of hardcode versions, generate it dynamically from inside Makefile and `.python-version`?